developer/contributing.rst

```diff
@@ -158,18 +158,16 @@ OpenWISP follows `PEP 8 -- Style Guide for Python Code
 <https://www.python.org/dev/peps/pep-0008/>`_ and uses the following
 automatic tools to check code conventions:
 
-- `flake8 <http://flake8.pycqa.org/en/latest/>`_ is used to
-  automatically check the quality of the python code being committed,
-  each python repo has either a ``flake8`` configuration defined in
-  ``setup.cfg`` or a ``runflake8`` script that you can launch with
-  ``flake8`` or ``./runflake8`` respectively.
 - `isort <http://isort.readthedocs.io/en/latest/>`_ is used in order
-  to sort import in a specifc predictable order; each python repo has
-  a ``runisort`` script that you can launch with ``./runisort``
-- Lastly, `black <https://black.readthedocs.io/en/stable/>`_ is used to
+  to sort import in a specifc predictable order.
+- `black <https://black.readthedocs.io/en/stable/>`_ is used to
```

Comment:
falke8, isort and black are all used, we need to mention this without mentioning the scripts which are not being used anymore.